content/team/greg-ball/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+name: "Greg Ball"
+description: "Staff Solution Architect"
+roles: ["author"]
+skills: ["Consulting", "Software Architecture", "Distributed Systems", "Domain Driven Design"]
+location: "Dallas, TX"
+hidden: false
+
+# Social Media
+twitter: ""
+website: ""
+facebook: ""
+linkedin: "gregory-ball-850b3481"
+github: ""
+pinterest: ""
+instagram: ""
+youtube: ""
+---
+
+Greg has spent his entire 15+ year career in the public sector space, which he continues to support as a Tanzu Labs consultant.  He loves to support customers in improving their practices and architectures in order to make software more reliable, more maintainable, and more fun for developers to work on.  When he's not working, family, mixology, and breadmaking are his passions. 
+
+<!-- more -->
```

Comment:
Greg,
Overall the article is pretty good. There could be some more conciseness in some places, but I didn't want to be marking it all up. Sean Scanlon turned me onto a tool called "Grammarly" that gives good suggestions and really helps clean up language. The paid version goes further with AI (although I'm still just using the free version). 

Beyond editorial comments, can you tell me what your goal was beyond what's already published on the topic? What additional thing are you bringing to the table? I'm not saying you aren't but I think it will be much more effective if you're very clear up front what motivated you to write the post. Like, yeah you can look up definitions for ubiquitous language, Eric Evans has written a bunch of stuff, but what I want you to know is XXXX.

Overall it's a really good article. I appreciate the time you put into thinking through it!

Warren